collect_env.py

```diff
@@ -512,6 +515,16 @@ def is_xnnpack_available():
     else:
         return "N/A"
 
+def get_env_vars():
+    env_vars = ''
+    secret_terms=('secret', 'token', 'api', 'access', 'password')
+    for k, v in os.environ.items():
+        if any(term in k.lower() for term in secret_terms):
+            v = '***'
+        if k not in environment_variables:
+            v = '***'
```

Comment:
this is too aggresive. let's only report:

- env vars in `environment_variables`
- env vars starting with `TORCH` or `NCCL` .

we can expand the list later as needed. sometimes, even exposing the name of env var is very risky.

Reply:
maybe with a variable named `report_prefix = ["TORCH", "NCCL"]` , and later we can just expand this list with one line change.

Reply:
Done